release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -136,13 +152,36 @@ xref:ocp-4-3-extend-user-workload-monitoring[extend user workload monitoring].
 [id="ocp-4-3-cluster-logging"]
 === Cluster logging
 
+[id="ocp-4-3-cluster-logging-log-forwarding"]
 ==== Cluster logging log forwarding
 
-The cluster logging log forwarding feature provides a way to ship container and node logs 
-to destinations that are not necessarily managed by the {product-title} cluster logging infrastructure. 
-Destination endpoints can be on or off your {product-title} cluster. Log forwarding provides an easier way to forward 
+The cluster logging log forwarding feature provides a way to ship container and node logs
+to destinations that are not necessarily managed by the {product-title} cluster logging infrastructure.
+Destination endpoints can be on or off your {product-title} cluster. Log forwarding provides an easier way to forward
 logs than the default log forwarding pipelines without requiring you to set the cluster to Unmanaged.
 
+[id="ocp-4-3-developer-experience"]
+=== Developer experience
+
+[id="ocp-4-3-odo"]
+==== OpenShift Do enhancements
+
+OpenShift Do (odo) has a few enhancements that focus more on the user experience
+of application deployment:
+
+* `PushTimeout` has been added as a configurable wait parameter.
+* Both the Service Catalog and component creation have been improved with
+extended output and information prompts.
+* Architecture support has been expanded to IBM Z and PowerPC platforms,
+binaries that are available for installation.
```

Comment:
Does this mean that support now provides installable binaries for IBM Z and PowerPC? If so, I'd suggest minor rewording to something like, "Architecture support has been expanded to IBM Z and PowerPC platforms, providing binaries that are available for installation" or "providing installable binaries."